addons/mail/tests/test_link_preview.py

```diff
@@ -170,3 +170,18 @@ def test_link_preview_no_content_type(self):
             url = self.source_url
             session = requests.Session()
             link_preview.get_link_preview_from_url(url, session)
+
+    def test_link_preview_ignore_list(self):
+        with patch.object(requests.Session, 'get', self._patch_with_og_properties), patch.object(requests.Session, 'head', self._patch_head_html):
+            ignore_list = [(False, 1), (self.source_url, 0), (".*\\.nothing", 0)]
+            for (ignore, counter) in ignore_list:
+                with self.subTest():
+                    self.env["ir.config_parameter"].sudo().search([("key", "=", "mail.mail_link_preview_ignore_list")]).unlink()
+                    if ignore:
+                        self.env["ir.config_parameter"].sudo().set_param("mail.mail_link_preview_ignore_list", ignore)
+                    message = self.test_partner.message_post(
+                        body=Markup(f'<a href="{self.source_url}">Nothing link</a>'),
```

Comment:
I find using `self.source_url` makes the test hard to understand and with little functional significance.
I'd rather see a few specifically chosen cases, like similar URL that we would have on the prod being filtered by the automatically generated regex, and a few other manually chosen one that make sense.

Reply:
we can now use any url we want in the test